linux_os/guide/system/auditing/auditd_configure_rules/audit_unsuccessful_file_modification/audit_rules_unsuccessful_file_modification_open_o_creat/rule.yml

```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Record Unauthorized Creation Attempts to Files (unsuccessful) - open with O_CREAT'
```

Comment:
'Record Unauthorized Creation Attempts to Files (unsuccessful) - O_CREAT'

Reply:
@redhatrises This rule will have similar check and prose for `openat` and `open_by_hande_at` syscalls. It would be good to have the syscall mentioned in title to be able to differentiate them from each other.